### DISCUSSION TITLE
Can only create new document links

### DISCUSSION THREAD

**Discussion by gboxxx** [outline/outline#4852] [Self Hosting] [answered]
I can't seem to get other documents to populate when I go to link to them. The only option I get is to create a new doc (see screenshot). Running this as an Admin and am associated with all the files.

Anyone else run into this? 

![image](https://user-images.githubusercontent.com/96701009/217923594-c7163915-f713-4158-91c8-2d6f586d3f83.png)

**Comment by tommoor** [comment, marked as answer]
It's a bug in the latest release. It's fixed on `main` but a new release hasn't happened yet.

**Reply by gboxxx** [reply]
Thanks, Tom!
